docs/index.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: default
+title: Home
+---
+
+This workflow uses several programs to call somatic variants and 
+aggregate the results.
+- strelka
+- mutect2
+- vardict
+- manta
+- somaticseq
+
+This workflow is part of [BioWDL](https://biowdl.github.io/)
+developed by [the SASC team](http://sasc.lumc.nl/).
```

Comment:
I don't think this url work anymore, actually...

Reply:
Good point. Will remove this.